lib/Bridge/TolerantParser/Reflection/TypeResolver/DeclaredMemberTypeResolver.php

```diff
@@ -2,14 +2,29 @@
 
 namespace Phpactor\WorseReflection\Bridge\TolerantParser\Reflection\TypeResolver;
 
+use Microsoft\PhpParser\Node\QualifiedName;
 use Microsoft\PhpParser\Token;
 use Phpactor\WorseReflection\Core\Type;
 use Phpactor\WorseReflection\Core\ClassName;
 use Microsoft\PhpParser\Node;
 
 class DeclaredMemberTypeResolver
 {
-    public function resolve(Node $tolerantNode, $tolerantType = null, ClassName $className = null): Type
+    private const WRONG_QUALIFIED_NAME = [
```

Comment:
should be plural, maybe rename to `RESERVED_NAMES`?

Reply:
Done